packages/webpack-cli/lib/plugins/InteractiveModePlugin.js

```diff
@@ -0,0 +1,207 @@
+const readline = require("readline");
+const { red, green, cyanBright, bold, gray } = require("colorette");
+const { SyncHook } = require("tapable");
+const logger = require("../utils/logger");
+
+let version;
+try {
+    version = require("webpack").version;
+} catch (err) {
+    process.exit(2);
+}
```

Comment:
Let's do not read webpack here, we support only v5 so `compiler.webpack` should be used

Reply:
Okay, on it 👍🏽